### DISCUSSION TITLE
Failed to find tag v1.0.0

### DISCUSSION THREAD

**Discussion by Blue-Marvel** [nexus-xyz/nexus-zkvm#298] [General]
What is the latest version cause I keep getting this error when I try to install using this command :

`cargo install --git https://github.com/nexus-xyz/nexus-zkvm nexus-tools --tag 'v1.0.0'
`
I got this command from the docs: [https://github.com/mztacat/nexus-zkvm/blob/main/README.md](url)
this is the error message:

`    Updating git repository [https://github.com/nexus-xyz/nexus-zkvm](url)
 error: failed to find tag \'v1.0.0\`

Caused by:
  reference 'refs/remotes/origin/tags/v1.0.0' not found; class=Reference (4); code=NotFound (-3)
`

**Comment by sjudson** [comment]
There is no tag `v1.0.0`, the highest tag is `v0.2.4` (see https://github.com/nexus-xyz/nexus-zkvm/tags).
